test/system/reports_test.rb

```diff
@@ -688,4 +688,9 @@ def wait_for_watch_change
     assert_text 'おめでとう！'
     assert_text '100日目の日報を提出しました。'
   end
+
+  test 'mentors can see reports　page if products published_at is nil' do
+    visit_with_auth "/reports/#{reports(:report18).id}", 'mentormentaro'
+    assert_text '2020年06月01日 の日報'
+  end
```

Comment:
```suggestion
  test 'mentors can see reports page if products published_at is nil' do
    visit_with_auth '/reports/unchecked', 'mentormentaro'
    click_link 'テストのnippou'
    assert_text '2020年06月01日 の日報'
    assert_text '今日は1時間学習しました。'
  end  
```

@clio209 さん、一度approveさせてもらったのですが、2点確認をお願いします！
- テスト名に全角スペースが含まれていたので半角スペースに修正をお願いします！（後付けですいません💦）
- テストのところを上記のように、日報一覧からpublished_atがnilの提出物を持つ人の日報をクリックして日報が表示される方がテスト内容が明確になるので良いのかと思いました…！
あくまで個人的な一意見なのでご判断は@clio209 さんにお任せします🙇‍♀️
ご確認お願いします🙏

Reply:
@saeyama さん
お忙しい中、見ていただき、誠に有難うございます。
suggestいただいた内容に修正いたしました🙇‍♂️